packages/pp-binary-ints/pp-binary-ints.0.1.0/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+synopsis: "Pretty Printing Binary Integers"
+description: """
+Pretty Printing Binary Integers
+This library contains functions for pretty printing integers as
+unsigned binary integers.
+"""
+maintainer: ["Ifaz Kabir"]
+authors: ["Ifaz Kabir"]
+license: "CC0-1.0"
+tags: ["printf" "format" "boolean" "binary"]
+homepage: "https://github.com/ifazk/pp-binary-ints"
+doc: "https://ifazk.github.io/pp-binary-ints/"
+bug-reports: "https://github.com/ifazk/pp-binary-ints/issues"
+depends: [
+  "dune" {>= "2.9"}
+  "ocaml" {>= "4.10" & < "4.14"}
```

Comment:
What is the reason for this restriction? Why does it not work with 4.14?

Reply:
I just haven't tested it for 4.14. On second thought, there's actually no good reason for this, do you want me to remove it?

Reply:
Yes that would be easier for you (avoiding to do manual changes everytime there is a new release) and us opam-repository maintainers. We check new versions of the compiler against all opam packages before it is release on a regular bases and you will be notified if it ever breaks some point

Reply:
Thanks for pushing the change. This is good to know for the future.